client/config/environment.js

```diff
@@ -34,6 +35,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
 
+    ENV['osfHostUrl'] = 'http://staging-api.osf.io';
```

Comment:
Should this be here? `ENV['osfHostUrl']` is also set to `'http://staging-api.osf.io'` on line 56